github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-tekton-tasks/tekton-tasks-periodics.yaml

```
@@ -0,0 +1,34 @@
+periodics:
+  - name: periodic-update-libguestfs-container-version
+    cron: 0 2 * * MON
```

nit: the rest of the periodics use the number notation for the day of the week

```suggestion
    cron: 0 2 * * 1
```

---

fixed